astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -25,7 +25,7 @@
 FREETYPE_261 = ft_version == Version("2.6.1")
 TEX_UNAVAILABLE = not matplotlib.checkdep_usetex(True)
 
-MATPLOTLIB_GT_3_4_3 = Version(matplotlib.__version__) > Version('3.4.3 ')
+MATPLOTLIB_DEV = Version(matplotlib.__version__).is_devrelease
```

Comment:
@pllim, you're more familiar with the history of this problem. Are only dev releases problematic, or have pre-releases likewise had ``dpi`` attribute weirdness. Looking at 
https://packaging.pypa.io/en/latest/version.html?highlight=is_devrelease#packaging.version.Version.is_prerelease
I see there's a flag ``.is_prerelease`` that is True for both pre and dev releases.

Reply:
Not sure. But we never test prerelease in the CI. If you look at `tox.ini`, it either installs stable versions or from source. 🤷

Reply:
Fair enough. Can always be adjusted later if we ever test for pre-releases.